data/Client_MobileClient

```diff
@@ -0,0 +1,8 @@
+name: "MobileClient"
+user_journeys {
+  name: "Play a Game"
+  dependencies {
+    target_service_name: "APIServer"
+    target_endpoint_name: "StartGame"
+  }
+}
```

Comment:
Should we commit the mock proto value files to source control? Or force users (devs) to generate them manually?

Reply:
it might be convenient to keep here as an example of mock data. 
having it in something like example/data is probably fine.